tests/test_suite.py

```diff
@@ -16,48 +17,89 @@
 from .conftest import show_widget
 
 
-@show_widget
-def test_suite(device):
-    device.name = 'test'
+@pytest.fixture(scope='function')
+def suite(qtbot, device):
     suite = TyphonSuite.from_device(device)
+    qtbot.addWidget(suite)
+    return suite
+
+
+@show_widget
+def test_suite_with_child_devices(suite, device):
     assert device in suite.devices
-    # Grab all device displays
-    assert len(suite._device_group.childs) == 1
-    child_displays = suite._device_group.childs[0].childs
+    device_group = suite.top_level_groups[0]
+    assert len(device_group.childs) == 1
```

Comment:
Is it the right time to jettison the `childs` flotsam and rename to children?

Reply:
Unfortunately this part of the `pyqtgraph` API not mine 😢